grunt/webpack-dev-server.js

```diff
@@ -5,15 +5,19 @@ let clone = require('clone');
 // clone the webpack config to separate configuration of webpack and dev server
 let webpackConfig = clone(require('./webpack').options);
 
-//port for webpack-dev-server
+// port for development server
 const port = +('GA'.split('').reduce((p, c)=> p + c.charCodeAt(), ''));
 
-// enable live reload without a script tag
-webpackConfig.entry.vendor.unshift(`webpack-dev-server/client?http://localhost:${port}`);
```

Comment:
Thanks for explaining this in-person, I'm going to leave a note here for myself:

> There is no inline: true flag in the webpack-dev-server configuration

This line from the docs is misleading if you think that "webpack-dev-server configuration" is what we're doing to configure the grunt task. It's not.

Again, `inline: true` works because we are not writing a webpack-dev-server configuration, we're writing a grunt task configuration.